components/Alumni/Awardees.js

```diff
@@ -0,0 +1,371 @@
+        {'name': 'Wareesha Ali', 'field': 'Life Sciences / Biotechnology'},
+        {'name': 'Waseem Ahmad', 'field': 'Engineering / Mechanical'},
+        {'name': 'Yasha Taimuri', 'field': 'Life Science / Zoology'},
+        {'name': 'Zainab Faiyaz', 'field': 'Life Science / Biotechnology'},
+        {'name': 'Zanbaz Ahmed Khan', 'field': 'Social Science / Economics'},
+        {'name': 'Zishan Hassan', 'field': 'Management / Business Administration'}
+      ]
+    },
+    { session: '2021-22',
+      awardees: [
+        {'name': 'Aafiya', 'field': 'Physics'},
+        {'name': 'Ayush Agrawal', 'field': 'Mechanical Engineering'},
+        {'name': 'Sehrish Akhtar', 'field': 'Physics'},
+        {'name': 'Sehrish Akhtar', 'field': 'Physics'},
+        {'name': 'Danish Alam', 'field': 'Physics'},
+        {'name': 'Danish Alam', 'field': 'Physics'},
+        {'name': 'Ayesha Nasir Alavi', 'field': 'Law'},
+        {'name': 'Mohammad Anas', 'field': 'Computer Engineering'},
+        {'name': 'Bushra Ansari', 'field': 'Biochemistry'},
+        {'name': 'Mohd Mushfique Javed Ansari', 'field': 'Architecture'},
+        {'name': 'Samina Irshad Ansari', 'field': 'Psychology'},
+        {'name': 'Heena Aslam', 'field': 'Electrical Engineering'},
+        {'name': 'Mohd Abdul', 'field': 'Baseer Physics'},
+        {'name': 'Mohammad Fahad', 'field': 'Electrical Engineering'},
+        {'name': 'Aquib Faraz', 'field': 'Commerce'},
+        {'name': 'Ifrah Farid', 'field': 'Biochemistry'},
+        {'name': 'Harshul Gupta', 'field': 'Physics'},
+        {'name': 'Arsalan Hameed', 'field': 'Computer Engineering'},
+        {'name': 'Saman Jafri', 'field': 'Biotechnology'},
+        {'name': 'Faisal Jamal', 'field': 'Biochemistry'},
+        {'name': 'Faraha Javed', 'field': 'Medicine'},
+        {'name': 'Shahrukh Khan', 'field': 'Electrical Engineering'},
+        {'name': 'Tayyibah Khanam', 'field': 'Electrical Engineering'},
+        {'name': 'Yusra Meraj Electronics', 'field': 'Engineering'},
+        {'name': 'Madiha Noman', 'field': 'English'},
+        {'name': 'Kunwar Mohammed Saaim', 'field': 'Computer Engineering'},
+        {'name': 'Saba Sarwar', 'field': 'Computer Science'},
+        {'name': 'Md. Showgat Jahan Shourave', 'field': 'Economics'},
+        {'name': 'Mohd Talha', 'field': 'Physics'},
+        {'name': 'Amber Tanweer', 'field': 'Law'}
+      ]
+    }
+  ]
+
+  return (
+    <div id="Awardees">
+      <h1 className="my-8 mx-4 sm:mx-12 lg:mx-20 bg-blue-850 lg:text-3xl text-2xl text-white text-center font-extrabold py-2 rounded-tl-3xl rounded-br-3xl">
+        Awardees
+      </h1>
+      <div className="flex flex-wrap justify-around my-8 mx-8 sm:mx-16 lg:mx-24 bg-gray-200">
+        {awardees.map((session, index) => (
+          <h3
+            className={`text-red-850 ${currentSession == session.session ? "bg-blue-850": ""} font-bold w-32 text-center text-xl px-4 py-2 cursor-pointer`}
```

Comment:
The blue background isn't looking good there. I would suggest discussing this with the design team.